### DISCUSSION TITLE
Load tips

### DISCUSSION THREAD

**Discussion by sunnamed434** [SmartlyDressedGames/Unturned-3.x-Community#4376] [Ideas]
While now those tips are outdated or useless, why not just make them editable from the server?

![image](https://github.com/SmartlyDressedGames/Unturned-3.x-Community/assets/65300126/2f2f0d55-b8c4-47a9-913c-0c0e1760bfb2)

**Comment by GazziFX** [comment]
It will be epileptic hell like in Rust

**Reply by sunnamed434** [reply]
Why and how does it work in Rust since I'm not familiar with Rust, I played for about 10-20 hours in total

**Reply by Jdance-Media** [reply]
There was a similar comment made by Butter about wanting to have server-side tips.  Maybe people would in fact find them useful. Especially considering how servers have different rules, plugins, mods, etc. They could actually have tips relevant to their specific server.

**Reply by Jdance-Media** [reply]
> It will be epileptic hell like in Rust

Could just disable rich text.
